### ISSUE TITLE
 cannot get related text

### ISSUE THREAD

**Issue by FaRzad-845** [codelucas/newspaper#901]
**Describe the bug**
When I receive information from a site like the one below, it cannot recognize the main part of the text ....

http://www.jahannews.com/news/769201

**To Reproduce**
```
url = "http://www.jahannews.com/news/769201"
article = Article(url, language='fa')
article.download()
article.parse()

print(article.text)
```

**Log**
Return ""

**Versions (please complete the following information):**
 - OS: [e.g. Linux Ubuntu 20.04]
 - Python Version 3.6
 - news-please Version latest

**Comment by johnbumgarner** [comment]
I'm looking into this issue, but I have no affiliation with this project.   The owner of this project won't respond to any inquiries about the project's status.  So I'm unsure that I can solve your issue, because this project is on life support unless the owner opens the project up for others to contribute more freely.

**Comment by jiangxinke** [comment]
yes, it seems that it can not hold some keywords in news website, or im not finding it yet
